### ISSUE TITLE
Add contribution guidelines

### ISSUE THREAD

**Issue by digitalcraftsman** [gohugoio/hugoDocs#1]
It's basically of the contribution guidelines in the main repo. Should also explain the purpose of the three branches.

**Comment by bep** [comment]
Also, even if the commit message prefixing does not make much sense, we should keep some kind of standard here as well. A sloppy commit log looks like a sloppy project.

**Comment by rdwatters** [comment]
I agree. I can propose a handful of new doc-specific prefixes, @bep @digitalcraftsman

**Comment by bep** [comment]
I was making a point of getting rid of the prefix, so no new artificial prefixes. It does not make sense in a small doc project as this is. It made sense when it was part of the Hugo codebase.

**Comment by rdwatters** [comment]
Okay, then perhaps I can help with editorial guidelines or some sort of commit standards if it helps.

**Comment by rdwatters** [comment]
And please note...

https://hugodocs.info/contribute/documentation/

**Comment by digitalcraftsman** [comment]
> Also, even if the commit message prefixing with "docs:"does not make much sense with one project, so we should get rid of that here

That makes sense - but old habbits. Release notes always had a 'docs' section. Now, the commits would have to be retrieved from the Git log of the `docs` submodule. Do you plan to keep this section in the docs
